schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	name CHAR(100),
```

Comment:
- [ ] Let's use VARCHAR for the name instead of CHAR. CHAR is fixed length and VARCHAR is variable length. CHAR always uses the same amount of storage space per entry, while VARCHAR only uses the amount necessary to store the actual text.